returnn/training.py

```diff
@@ -298,7 +297,10 @@ class EpochData:
 
             try:
                 with open(file_path, "rt") as file:
-                    return eval(file.read().strip(), {"EpochData": EpochData, "np": np})
+                    return eval(
+                        file.read().strip(),
+                        {"EpochData": EpochData, "nan": float("nan"), "inf": float("inf"), "np": np},
```

Comment:
Is it ok to just have the `np` key here as this is the common name under which numpy is imported, or should we also add `"numpy": np`?

Reply:
It's mandatory, because this is how Numpy `float64.__repr__` prints it:
```
In [1]: import numpy

In [2]: numpy.float64(1.23)
Out[2]: np.float64(1.23)
```

Reply:
Ahh, I see. Then the other variant also won‘t be necessary.

Reply:
(Btw, e.g. `numpy.array(1.23)` has just the repr `array(1.23)`, so maybe adding `array` might also make sense, but not here, or not now, as we never have arrays in the LR file.)